src/inputs/RichTextField.tsx

```diff
@@ -130,22 +119,63 @@ export function RichTextField(props: RichTextFieldProps) {
     }
   }, [value, readOnly]);
 
+  useEffect(() => {
+    if (parentRef.current) {
+      function onEditorInit() {
+        editor.current = (editorElement.current as any).editor;
+        if (mergeTags !== undefined) {
+          attachTributeJs(mergeTags, editorElement.current!);
+        }
+
+        currentHtml.current = value;
+        editor.current!.loadHTML(value || "");
+      }
+
+      parentRef.current.addEventListener("trix-initialize", onEditorInit);
```

Comment:
What if, on the first render, we attached this directly to `window`? Like:

* Use `useMemo` with `deps=[]` to get a unique-ish id (to handle two RichTextFields initing during the same render)
* Use the same `useMemo` to hook up a `window.trix-initialize` callback 
* Use the unique-ish id in the call to `createElement`
* When the callback is fired, make sure the event is for our unique-ish id, save the editor to a ref, and unregister our `window.trix-initiatize` callback?

Reply:
Would be better than the rerender solution I have now. Will give it a shot. Thanks for the suggestion.

Reply:
Updated! Thanks for the suggestion